pkgs/development/python-modules/qubes-core-admin/0005-fix-data_percent-might-be-empty.patch

```diff
@@ -0,0 +1,56 @@
+From 3d56e8b0f177f98914cecbe0d45afefa22c952ec Mon Sep 17 00:00:00 2001
+From: Yaroslav Bolyukin <iam@lach.pw>
+Date: Mon, 16 Sep 2024 18:54:21 +0200
+Subject: [PATCH 5/6] fix: data_percent might be empty
```

Comment:
TODO: Submit this patch upstream.

It is possible to include necessary changes in nixos module, but those changes may break workflows of some users.

Reply:
This patch doesn't work well.
Resulting errors might be not good, but I think it can be solved with better documentation/error message clarificatios.